assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -165,8 +165,8 @@ cmdline_additionalArg=+{{.AdditionalArgs}}
 
 {{if .PerPodPowerManagement}}
 cmdline_pstate=+intel_pstate=passive
-{{end}}
-
-{{if .HighPowerConsumption}}
+{{else}}
+{{if .RealTimeHint}}
```

Comment:
I am not entirely happy with how this nested condition looks like, I will not block on it, but I would prefer something more readable if possible.

Reply:
done, I didn't realize that it was else if in go templates